src/main/java/usecases/CourseEnrollmentUseCase/CourseEnrolmentInteractor.java

```diff
@@ -16,13 +16,6 @@ public CourseEnrolmentInteractor(EnrolmentCourseDataAccess courseDbGateway,
         this.enrolmentPresenter = enrolmentPresenter;
     }
 
-    /**
-     * calls CourseDbGateway to load course from Course Database
-     */
-    public void callToRetrieveCourse(EnrolmentDbRequestModel dbRequestModel) {
-        courseDbGateway.retrieveCourse(dbRequestModel);
-    }
-
```

Comment:
I agree with this change ! I think that since the method only performs one function call, it might make more sense to have that call in the enrol method instead of having another method for that.